src/__test__/CryptoList.test.js

```diff
@@ -0,0 +1,14 @@
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import store from '../redux/store';
+import CryptoList from '../pages/CryptoList';
+
+/* eslint-disable comma-dangle */
+it('renders CryptoList Component correctly', () => {
+  const tree = renderer.create(
+    <Provider store={store}>
+      <CryptoList />
+    </Provider>
+  );
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] This test is failing. Let's fix it

<img width="739" alt="Screenshot 2022-11-23 at 2 17 24 PM" src="https://user-images.githubusercontent.com/40503704/203556646-961d63ca-bf92-433d-bc2b-4fe513d92fe2.png">